arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -0,0 +1,679 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MediaServicesManagementClient",
+    "description": "Media Services resource management APIs.",
+    "version": "2015-10-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices": {
+      "get": {
+        "summary": "List all of the Media Services in a resource group.",
+        "operationId": "MediaServices_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "Name of the resource group within the Azure subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],        
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MediaServiceCollection"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        }

```

Comment:
MediaSeriveCollection has a nextLink so is this pageable? If so, consider extension x-ms-pageable.  
See https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-pageable